third_party/mbedtls/hardware_entropy.c

```diff
@@ -1,5 +1,5 @@
 /*
- *    Copyright 2016 Nest Labs Inc. All Rights Reserved.
+ *    Copyright 2016 The OpenThread Authors. All Rights Reserved.
```

Comment:
@jwhui @beriberikix  Should the copyright on this file be changed? This is in the third_party directory, so is it by the OpenThread authors or is it by Nest?